link.json

```diff
@@ -0,0 +1,194 @@
+{
+    "appid": "c5a92c45-8c3e-417b-be46-01e8d7c677a6",
+    "name": "Link",
+    "description": "Generates a widget with clickable links",
+    "publisher": "Mhike",
+    "type": "devops",
+    "main_module": "link_connector.py",
+    "app_version": "1.0.0",
+    "utctime_updated": "2022-06-12T14:23:35.606945Z",
+    "package_name": "phantom_link",
+    "product_vendor": "Mhike",
+    "product_name": "Link",
+    "product_version_regex": ".*",
+    "min_phantom_version": "4.9.0",
```

Comment:
The static test is failing due to min_phantom_version value.
Is there any reason why we haven't modified this?

Reply:
Yes, @zamastyle intended the min phantom version as 4.9.0 instead of the latest 5.2.0 version. Hence, I didn't change it.